pgt/src/PGT/Prelude.hs

```diff
@@ -1,4 +1,11 @@
-module PGT.Prelude (module Exports) where
+module PGT.Prelude
+  ( module Exports
+  , showc
+  )
+where
 
-import Data.Conversions as Exports
-import MPrelude         as Exports
+import Data.Conversions    as Exports
+import MPrelude            as Exports
+
+showc :: forall b a . (Show a, Conversion b String) => a -> b
+showc = convert . show
```

Comment:
__question:__ Not a blocker, but this appears in  multiple projects. Should it be moved to `MPrelude` eventually?